src/frontend/context/SimuleringContext.tsx

```diff
@@ -130,23 +117,27 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
         return totaltAvvik <= maksgrenseForAvvikIBeløpVedMigrering;
     };
 
-    const harStoppetMigreringAvvikInnenforBeløpsgrenser =
-        erMigreringMedStoppISimulering &&
-        harKunNegativeEllerKunPositiveAvvik(perioderesultaterFørMars2023) &&
+    const behandlingErMigreringMedAvvikInnenforBeløpsgrenser =
+        erMigreringFraInfotrygdMedAvvik &&
         harMaks1KroneIAvvikPerBarn(perioderesultaterFørMars2023) &&
         harTotaltAvvikUnderBeløpsgrense(perioderesultaterFørMars2023);
 
+    const behandlingErMigreringMedAvvikUtenforBeløpsgrenser =
+        erMigreringFraInfotrygdMedAvvik &&
+        (!harMaks1KroneIAvvikPerBarn(perioderesultaterFørMars2023) ||
+            !harTotaltAvvikUnderBeløpsgrense(perioderesultaterFørMars2023));
```

Comment:
Lurer på om det kunne vært forenklet litt:
```suggestion
        !behandlingErMigreringMedAvvikInnenforBeløpsgrenser;
```

Reply:
Veldig godt poeng!
Kjører på med det istedenfor 😊